commands/create-account.js

```diff
@@ -0,0 +1,50 @@
+
+const exitOnError = require('../utils/exit-on-error');
+const connect = require('../utils/connect');
+const { KeyPair, utils } = require('nearlib');
+
+module.exports = {
+    command: 'create_account <accountId>',
+    desc: 'create a new developer account',
+    builder: (yargs) => yargs
+        .option('accountId', {
+            desc: 'Unique identifier for the newly created account',
+            type: 'string',
+            required: true
+        })
+        .option('masterAccount', {
+            desc: 'Account used to create requested account.',
+            type: 'string',
+            required: true
+        })
+        .option('publicKey', {
+            desc: 'Public key to initialize the account with',
+            type: 'string',
+            required: false
+        })
+        .option('initialBalance', {
+            desc: 'Number of tokens to transfer to newly created account',
+            type: 'string',
+            default: '0.1'
```

Comment:
why we change it to 0.1?

Reply:
So that if you have created your master account in wallet, you can create multiple accounts in shell without completely draining master account balance (it's 10 NEAR by default).

Reply:
I think it is not a good way to do so. give more instructions to user what `create_account` command is from and how to use it make it more clear.

Reply:
do you suggest not having default amount at all? or different default amount?

Reply:
maybe not having default amount but required amount from user is better. since 0.1 or even less is not good number to guess how many account user want.

Reply:
> maybe not having default amount but required amount from user is better.

I agree there is some trade off between discoverability and convenience here. However I highly prefer tools where you can succeed most of the time with reasonable details. We need to make sure to have `--initialBalance` example in docs though /cc @amgando